ownCloud/Server List/ServerListTableViewController.swift

```diff
@@ -300,72 +302,72 @@ class ServerListTableViewController: UITableViewController, Themeable {
+					alertController.addAction(UIAlertAction(title: "Delete".localized, style: .destructive, handler: { (_) in
 
-								vault.erase(completionHandler: { (_, error) in
-									OnMainThread {
-										if error != nil {
-											// Inform user if vault couldn't be erased
-											let alertController = UIAlertController(title: NSString(format: "Deletion of '%@' failed".localized as NSString, bookmark.shortName as NSString) as String,
-																message: error?.localizedDescription,
-																preferredStyle: .alert)
+						self.lockedBookmarks.append(bookmark)
 
-											alertController.addAction(UIAlertAction(title: "OK".localized, style: .default, handler: nil))
+						OCCoreManager.shared.scheduleOfflineOperation({ (bookmark, completionHandler) in
+							let vault : OCVault = OCVault(bookmark: bookmark)
 
-											self.present(alertController, animated: true, completion: nil)
-										} else {
-											// Success! We can now remove the bookmark
-											self.ignoreServerListChanges = true
+							vault.erase(completionHandler: { (_, error) in
+								OnMainThread {
+									if error != nil {
+										// Inform user if vault couldn't be erased
+										let alertController = UIAlertController(title: NSString(format: "Deletion of '%@' failed".localized as NSString, bookmark.shortName as NSString) as String,
+																				message: error?.localizedDescription,
+																				preferredStyle: .alert)
 
-											OCBookmarkManager.shared.removeBookmark(bookmark)
+										alertController.addAction(UIAlertAction(title: "OK".localized, style: .default, handler: nil))
 
-											tableView.performBatchUpdates({
-												tableView.deleteRows(at: [indexPath], with: UITableView.RowAnimation.fade)
-											}, completion: { (_) in
-												self.ignoreServerListChanges = false
-											})
+										self.present(alertController, animated: true, completion: nil)
+									} else {
+										// Success! We can now remove the bookmark
+										self.ignoreServerListChanges = true
 
-											self.updateNoServerMessageVisibility()
-										}
+										OCBookmarkManager.shared.removeBookmark(bookmark)
 
-										if let removeIndex = self.lockedBookmarks.index(of: bookmark) {
-											self.lockedBookmarks.remove(at: removeIndex)
-										}
+										tableView.performBatchUpdates({
+											tableView.deleteRows(at: [indexPath], with: UITableView.RowAnimation.fade)
+										}, completion: { (_) in
+											self.ignoreServerListChanges = false
+										})
 
-										completionHandler()
+										self.updateNoServerMessageVisibility()
 									}
-								})
-							}, for: bookmark)
-						}))
 
-						self.present(alertController, animated: true, completion: nil)
-					}
-				}),
+									if let removeIndex = self.lockedBookmarks.index(of: bookmark) {
+										self.lockedBookmarks.remove(at: removeIndex)
+									}
```

Comment:
correct indentation in this file?

Reply:
@jesmrec I re-indented the file using following Xcode settings. Should be fine
<img width="548" alt="screenshot 2019-02-14 at 15 11 18" src="https://user-images.githubusercontent.com/228537/52792139-eea57c00-306a-11e9-89da-590d6684e8d9.png">